### PR TITLE
docs: add information about Griot

### PR DESCRIPTION
**Pull request by CBID2**
## Description
I noticed that there was no category for Caribbean food, so I created one and added information about Griot . 

## What type of PR is this? (check all applicable)

- [ ] ☝️ Add a pizza fact or trivia
- [ ] 🧑‍🍳 Add a pizza recipe
- [ ] 🗺️ Add a regional pizza
- [x] 🍴 Add a traditional food
- [ ] 📝 Documentation Update

## Related Tickets & Documents
Related to #86 

## Added to documentation?

- [ ] 📜 README.md
- [x] 🙅 no documentation needed

### CODE REVIEW

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @CBID2,

Thank you for this addition! ✨ 

As I'm not too familiar in which continent Caribbean is located, I googled it. Most resources — I link to some of them below for your reference — say that it's part of North America.

- https://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_in_North_America
- https://visaguide.world/north-america-caribbean/
- https://www.quora.com/Is-the-Caribbean-a-part-of-North-or-South-America

That said, can you please list your addition under North America?

Thanks. 🙂

**Comment by CBID2** [comment]
> Hey @CBID2,
> 
> 
> 
> Thank you for this addition! ✨ 
> 
> 
> 
> As I'm not too familiar in which continent Caribbean is located, I googled it. Most resources — I link to some of them below for your reference — say that it's part of North America.
> 
> 
> 
> - https://en.wikipedia.org/wiki/List_of_sovereign_states_and_dependent_territories_in_North_America
> 
> - https://visaguide.world/north-america-caribbean/
> 
> - https://www.quora.com/Is-the-Caribbean-a-part-of-North-or-South-America
> 
> 
> 
> That said, can you please list your addition under North America?
> 
> 
> 
> Thanks. 🙂 

Made the change @adiati98! 😊

File: world-traditional-food.md [hunk 1]
```diff
@@ -19,18 +13,34 @@ If your hometown or country has a traditional food (especially if it's your favo
 
 ## Table of Contents
 
+- [Table of Contents](#table-of-contents)
 - [North America](#north-america)
+  - [Caribbean](#caribbean)
```
Comment (adiati98) [inline world-traditional-food.md:18]: Because we want to list the countries, let's remove the Caribbean here.

File: world-traditional-food.md [hunk 2]
```diff
@@ -19,18 +13,34 @@ If your hometown or country has a traditional food (especially if it's your favo
...
+    - [Dosa](#dosa)
   - [Indonesia](#indonesia)
+    - [Rendang](#rendang)
   - [Philippines](#philippines)
+    - [Adobong Manok](#adobong-manok)
 - [Australia / Oceania](#australia--oceania)
 
 ## North America
 
+### Caribbean
```
Comment (adiati98) [inline world-traditional-food.md:35]: Let's remove this as well.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @CBID2,

I've left a couple of reviews here.

Also, we've merged a PR before yours. So, please resolve the conflicts before we can review your PR. Thanks! ✨

**Comment by CBID2** [comment]
Hey @adiati98. I made the changes

File: world-traditional-food.md [hunk 3]
```diff
@@ -1,9 +1,3 @@
-# Traditional Food Around the World
-
-Welcome to the Traditional Food Around the World section of our repo! 🗺️🍴
-
-If your hometown or country has a traditional food (especially if it's your favorite dish!), we encourage you to share it here. We're excited to learn about food culture from around the world!
-
```
Comment (adiati98) [inline world-traditional-food.md:6]: I'm not sure why you deleted these. We don't want to remove them. So, please add them back.

File: world-traditional-food.md [hunk 4]
```diff
@@ -19,18 +13,31 @@ If your hometown or country has a traditional food (especially if it's your favo
...
 ## North America
 
+### Haiti
+
+#### Griot
+
+Griot (pronounced "gree-oh") is a popular Haitian dish. It is pork marinated in a mixture of herbs and spices and fried in oil, resulting in a crispy and tender meat, and the sauce is rich and flavorful. Griot is often topped with a pickled vegetable sauce called "pikliz"(pronounced "pick-leez") and served along side with fried plantains, rice, and red beans, making it the national dish of Haiti.
+
+ <img src="./images/griot.jpg" width="400" alt="a plate of rice and beans with fried plantains and griot topped with pikliz">
 ## South America
```
Comment (adiati98) [inline world-traditional-food.md:41]: Add a space here.

```suggestion
 <img src="./images/griot.jpg" width="400" alt="a plate of rice and beans with fried plantains and griot topped with pikliz">
 
## South America
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
@CBID2 there are things to fix here.

Also, can you update your branch with the `main` branch?
We've merged a PR that added Indian Samosa, but I don't see it here.

Thanks! ✨

**Comment by CBID2** [comment]
> @CBID2 there are things to fix here.
> 
> Also, can you update your branch with the `main` branch? We've merged a PR that added Indian Samosa, but I don't see it here.
> 
> Thanks! ✨

Hey @adiati98. I made the changes.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @CBID2,

Can you please ask for review once you've addressed all feedback?
I see you quoted my comment, but you haven't addressed something there.

Thanks.

**Comment by CBID2** [comment]
> Hey @CBID2,
> 
> Can you please ask for review once you've addressed all feedback? I see you quoted my comment, but you haven't addressed something there.
> 
> Thanks.

Hey @adiati98. I reread your previous feedback. The information for the Samosa is on line 61. For some odd reason, GitHub won’t let me link it here.

**Comment by adiati98** [comment]
@CBID2, have you tried to pull it locally?

**Comment by CBID2** [comment]
> @CBID2, have you tried to pull it locally?

Yes @adiati98

**Review by adiati98 — COMMENTED** [review]
Hey @CBID2,

Thanks for the changes! ✨ 

FYI, I've removed some things as below:

- The foods that you've added to the Table of Contents in [this commit](https://github.com/open-sauced/pizza-verse/pull/100/commits/8023682ab9e9d380d41cedd56a90a4d60bad200d).
-  Duplication of image resource that somehow got added in [this commit](https://github.com/open-sauced/pizza-verse/pull/100/commits/31ee34ed7cff75a585ed62408c57aca3007cd511).

go time gif

**Review by adiati98 — APPROVED** [review]

**Comment by CBID2** [comment]
Yay! Now everybody gets to enjoy the awesomeness of Haitian food @adiati98! 😊

**Pull request merged**